README.md

```diff
@@ -14,6 +14,40 @@ programming language).
 currently built using TypeScript on top of [oclif](https://oclif.io/) to speed
 up development of the CLI tool.
 
+## Using with Docker
+
+To run `themis-contract` using Docker, you can build the Docker image for it
+yourself for now (we are working on getting up on Docker Hub):
+
+```bash
+# Clone the latest source
+git clone git@github.com:informalsystems/themis-contract.git
+cd themis-contract
+
+# Build a Docker image from the latest source you've just cloned
+docker build -t themis-contract:latest
+
+# Shows the "help" command for Themis Contract
+docker run --rm -it \
+  themis-contract:latest
+
+# Create a new contract from the specified template in the current directory
+# (assumes that the template.md file is in the current directory)
+docker run --rm -it \
```

Comment:
This failed for me with

```
[sf@comp informal-shares]$ sudo docker run --rm -it   -v `pwd`:/contracts   themis-contract:latest   new -t ./template.md
2020-05-28 15:20:28 info Created new contract: contract.toml
2020-05-28 15:20:28 error 
Your $EDITOR environment variable is not set.
Set it to the command/path of your editor in ~/.zshenv or ~/.bashrc:

  export EDITOR=atom
```

Reply:
I've tried 

```
sudo docker run -e EDITOR --rm -it -v `pwd`:/contracts   themis-contract:latest   new -t ./template.md
```

but this didn't work, despite the fact that

```
[sf@comp informal-shares]$ echo $EDITOR
/usr/bin/emacs
```

Reply:
It wouldn't work, because your `emacs` binary would need to be inside the container and would need to run within the container. Remember, `themis-contract` is running inside the container and can't execute applications outside of the container on your host machine.

Reply:
Ah, right. So we need the --noedit flag i guess?

Reply:
Sure!